lib/utils/html-fn.html

```diff
@@ -0,0 +1,61 @@
+<!--
+@license
+Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<link rel="import" href="boot.html">
+<script>
+  (function() {
+    'use strict';
+
+    /**
+     * @param {*} value Object to stringify into HTML
+     * @return {string} HTML stringified form of `obj`
+     */
+    function htmlValue(value) {
+      if (value instanceof HTMLElement) {
+        return /** @type {!HTMLElement} */(value).innerHTML;
+      } else if (value instanceof Node) {
+        return /** @type {!Node} */(value).textContent;
+      } else {
+        return value.toString();
```

Comment:
I think this should be `String(value)` - that'll handle `null`, `undefined`, and matches the template literal spec, afaik